packages/integration-sdk-testing/src/config.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  IntegrationInstanceConfig,
+  IntegrationInvocationConfig,
+} from '@jupiterone/integration-sdk-core';
+
+export interface StepTestConfig<
+  TInvocationConfig extends IntegrationInvocationConfig = IntegrationInvocationConfig
+> {
+  stepId: string;
+  invocationConfig: TInvocationConfig;
+}
+
+export interface StepTestConfigWithCredentials<
```

Comment:
The suffix `Credentials` isn't working for me, since the extension adds an `instanceConfig` property, not a `credentials` property. How about `StepTestConfigWithInstanceConfig`? Maybe `StepTestConfig` needs a different name. However, I'm coming up short on suggestions this late hour.

Reply:
Maybe we don't need both. Maybe we just define & export `StepTestConfig` and it includes all three properties.

Reply:
const config: StepTestConfig = {
  stepId: 'my-step-id',
  invocationConfig,
  instanceConfig,
};

const stepResult = await executeStepWithDependencies(config);

expect(stepResult).toMatchStepMetadata(config);